### ISSUE TITLE
Implement velocity calculation

### ISSUE THREAD

**Issue by sfalkena** [sfalkena/EPO-4_B05#1]
In order for KITT to brake better, it might help to implement a way to estimate the velocity. If this can be combined with an estimate for the delays, we can more accurately determine its stopping range

**Comment by sdelfos** [comment]
I would say the velocity is only regulated by the given motorspeed command, given that the car is finished accelerating and is traveling at constant speed, which will most probably the case when it reaches the obstacle. So this would mean that we just have to measure how fast the car rides at what motorspeed.

**Comment by sfalkena** [comment]
For the first exercise it probably is (although I have no idea about the range of the midterm test),

**Comment by sdelfos** [comment]
True. Also note that the max speed/acceleration is depend on the battery voltage. A not fully charged battery will deliver less voltage and therefore be not as fast. I've also quickly went through the next module and noticed that they require you to find solutions for these problems. It might not be needed to get this done before the mid-term.
